travel-event-host-app/src/components/header/Header.tsx

```diff
@@ -0,0 +1,146 @@
+'use client';
+import { useState } from 'react';
+import styles from './styles.module.css';
+import Link from 'next/link';
+import Avatar from '@mui/material/Avatar';
+import CircularProgress, { circularProgressClasses } from '@mui/material/CircularProgress';
+
+export default function Header() {
+  const [lang, setLang] = useState('EN');
```

Comment:
Suggestion: create an `enum` for the language. It can have only English for now, but we can add more later. Then type the useState
```suggestion
  const [lang, setLang] = useState<Language>(Language.En);
```